README.md

```diff
@@ -138,12 +138,27 @@ with threadpool_limits(limits=1, user_api='blas'):
 
 ### Known Limitations
 
-`threadpool_limits` does not act as expected in nested parallel loops
-managed by distinct OpenMP runtime implementations (for instance libgomp
-from GCC and libomp from clang/llvm or libiomp from ICC).
+- `threadpool_limits` can fail to limit the number of inner threads when nesting
+  parallel loops managed by distinct OpenMP runtime implementations (for instance
+  libgomp from GCC and libomp from clang/llvm or libiomp from ICC).
+
+  See the `test_openmp_nesting` function in [tests/test_threadpoolctl.py](
+  https://github.com/joblib/threadpoolctl/blob/master/tests/test_threadpoolctl.py)
+  for an example. More information can be found at:
+  https://github.com/jeremiedbb/Nested_OpenMP
+
+  Note however that this problem does not happen when `threadpool_limits` is
+  used to limit the number of threads used internally by BLAS calls that are
+  them-selves nested under OpenMP parallel loops as long as `threadpool_limits`
+  is called prior to entering the outer parallel OpenMP region. This works as
```

Comment:
I'm not sure this is true anymore, since the deadlock issues were due to the libiomp/libomp interactions on linux which we stopped supporting.

Reply:
I am not sure I understand.

The issue described at the beginning of this itemized section is still there. I get XFAILED for `test_openmp_nesting[2]` on my local Linux machine if I build the inner or outer loop with clang-8 and gcc for the other.

The OpenMP / BLAS case, tested via `check_nested_prange_blas`  works however. This is the case we have in scikit-learn. But then `threadpool_limit(1, user_api="blas")` is called prior to entering the outer parallel loop.

Reply:
I did not point only the line I was referring to :/
My remark was reffering to `is called prior to entering the outer parallel`. Putting apart the linux/llvm/intel case, it should not be an issue to call threadpoolctl inside the outer loop.

Reply:
Ok, I fixed this by removing this misleading/inaccurate information.